e2e/compose-env.yaml

```diff
@@ -1,9 +1,10 @@
 services:
+
   registry:
     image: 'registry:2'
 
   engine:
-      image: 'docker:${TEST_ENGINE_VERSION:-stable-dind}'
+      image: 'docker:${ENGINE_VERSION:-stable}-dind'
```

Comment:
The default value `stable-dind` is still 19.03. We should probably default to the latest release?